GPU/Common/SplineCommon.cpp

```diff
@@ -926,16 +926,19 @@ void DrawEngineCommon::SubmitSpline(const void *control_points, const void *indi
 	patch.patchFacing = patchFacing;
 
 	if (g_Config.bHardwareTessellation && g_Config.bHardwareTransform && !g_Config.bSoftwareRendering) {
-		float *pos = (float*)(decoded + 65536 * 18); // Size 3 float
-		float *tex = pos + count_u * count_v * 3; // Size 3 float
-		float *col = tex + count_u * count_v * 3; // Size 4 float
+		int stride = g_Config.iGPUBackend == GPU_BACKEND_VULKAN || g_Config.iGPUBackend == GPU_BACKEND_DIRECT3D11 ? 4 : 3;
```

Comment:
Oh, believe me, I've been bitten. True, it doesn't have a ternary, but try this:

     a & b == c & d

This is not what you'd expect, instead it evaluates to

    a & (b == c) & d

So anything that doesn't involve the common math operators, which do behave like you expect, I want written out with parenthesis. Personal preference though.